### DISCUSSION TITLE
Github actions does not start "Expected — Waiting for status to be reported"

### DISCUSSION THREAD

**Discussion by shamprasadrh** [community/community#25992] [Actions]
<p>We have PR that needs to be merged but CI doesn’t start and it is stuck in <strong>"Expected — Waiting for status to be reported"</strong></p>
<p>I have closed the PR and reopen the same but no luck.</p>

**Comment by Yanjingzhu** [comment]
<p>Could you please check the “Branch protection rules” in repo settings &gt; Branches tab? Have you checked that job under “Require status checks to pass before merging” section?<br>
There is a previous <a href="https://github.community/t/on-push-pull-request-paths-and-status-checks/18327">ticket</a> which reported that when use path filter for push or pull_request event, the job stuck on "Expected - Waiting " status.<br>
Does this match your scenario? If so, please remove the status check in branch protect rules .</p>

**Comment by shamprasadrh** [comment, marked as answer]
<p>yanjingzhu:</p>
<p>I didn’t wanted to remove the branch rules. Seems like issue is due to branch name had a ‘/’ in it after creating new branch without ‘/’ it works.</p>
